Makefile

```
@@ -1,4 +1,9 @@
-NAME    := argobots
-SRC_EXT := gz
+NAME           := argobots
+SRC_EXT        := gz
+GITHUB_PROJECT := pmodels/$(NAME)
+# Patch commit ID to apply
+# PKG_GIT_COMMIT :=
+# This list of files that are in the upstream git repo but are not included in upstream's releases
+# PATCH_EXCLUDE_FILES :=
```

Per my previous comments regarding this method only being acceptable for a large number of patches in a git delta and that such a large number of patches is not acceptable for _product_ packages, let's remove this method entirely from this file please.  Ultimately we should only be using this method for test RPMs that we do not distribute, and even then, only as necessary.

```suggestion
NAME    := argobots
SRC_EXT := gz
```

---

Fair enough, I will do the change.

- [ ] Remove useless local patch management

---

Fixed with commit 8779a83